drivers/nrf_802154/driver/src/nrf_802154_peripherals_nrf52.h

```diff
@@ -113,7 +112,7 @@ extern "C" {
  *
  */
 #ifndef NRF_802154_PPI_RADIO_RAMP_UP_TRIGG
-#define NRF_802154_PPI_RADIO_RAMP_UP_TRIGG NRF_PPI_CHANNEL6
+#define NRF_802154_PPI_RADIO_RAMP_UP_TRIGG 6
```

Comment:
```suggestion
#define NRF_802154_PPI_RADIO_RAMP_UP_TRIGG 6U
```
Please add `U` suffix to all `NRF_802154_PPI_xxxx` definitions you updated.
It is very likely that this file will be overwritten by a deploy from internal sdk-nrf-802154 repo. I'm introducing the changes there with `U` suffix.

Reply:
Done